
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<link rel="stylesheet" type="text/css" href="css/GENEDBQuery.css"/>
<link rel="stylesheet" type="text/css" href="css/IndividualResult.css"/>
<link type="text/css" href="css/GENEDBQuery_old.css" rel="stylesheet" />
<link type="text/css" href="css/styleFoot.css" rel="stylesheet" />
<title>IMGT/GENE-DB Consultation</TITLE>
</head>
<body id="GENE-DB">
<table style="width:100%;">
<tr>
<td align="left" nowrap valign="middle" width="70%">
<font size="+3" color="#000099"><b>THANK YOU</b><br/>
<b>for using <a href="/genedb/doc">IMGT/GENE-DB</a></b></font>
</td>
<td class="notshow" align=right valign=top width="30%"><a href="/"><img SRC="images/logo_IMGT2.png" ALT="WELCOME !" class="mainlogo"/></a>
</td>
</tr>
<tr>
<td class="logolegendleft">IMGT&reg;, the international ImMunoGeneTics information system&reg;</td>
<td class="logolegend">http://www.imgt.org</td>
</tr>
</table>
<hr/>
<img src="images/logoGene-DB.png" ALT="Gene-DB logo" class="mainlogo" />

<table class="citation">
<tr>
<td class="citation">
<b>Citing IMGT/GENE-DB</b>: Giudicelli, V. et al. Nucleic Acids Res., 33: D256 - D261 (2005).
PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/15608191/">15608191</a>
<a href="/PDF/NAR/33_D256-D261_2005.pdf"><img src="/images/pdficon.png" alt="pdf" /></a>
</td>
</tr>
</table>
<p class="programversion">IMGT/GENE-DB program version: <a href="/IMGTgenedbdoc/programversions.html">3.1.30</a> (18 September 2020)<br/>
<a href="/IMGTgenedbdoc/dataupdates.html">IMGT/GENE-DB data updates </a>
</p>

<h2 class="subtitle"><a href="/IMGTScientificChart/SequenceDescription/IMGTreferencesequences.html" >IMGT/GENE-DB reference sequences</a> in FASTA format:</h2>
<h2 class="subtitle">Amino acids  sequences with gaps according to the <a href="/IMGTScientificChart/Numbering/IMGTnumberingCR.html">IMGT unique numbering</a><br /> for <span class="casimir">F+ORF+in-frame P with IMGT Gaps</span> alleles including orphons</h2>
<hr>
<pre>
The FASTA header contains 15 fields separated by '|':

1. IMGT/LIGM-DB accession number(s)
2. IMGT gene and allele name
3. species
4. IMGT allele functionality
5. exon(s), region name(s), or extracted label(s)
6. start and end positions in the IMGT/LIGM-DB accession number(s)
7. number of nucleotides in the IMGT/LIGM-DB accession number(s)
8. codon start, or 'NR' (not relevant) for non coding labels
9. +n: number of nucleotides (nt) added in 5' compared to the corresponding label extracted from IMGT/LIGM-DB
10. +n or -n: number of nucleotides (nt) added or removed in 3' compared to the corresponding label extracted from IMGT/LIGM-DB
11. +n, -n, and/or nS: number of added, deleted, and/or substituted nucleotides to correct sequencing errors, or 'not corrected' if non corrected sequencing errors
12. number of amino acids (AA): this field indicates that the sequence is in amino acids
13. number of characters in the sequence: nt (or AA)+IMGT gaps=total
14. partial (if it is)
15. reverse complementary (if it is)
</pre>
<hr/>
<b>Number of results = 99</b><br />
<pre>
>NW_001095158|IGLV1-10*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.ASGAPGQSVT.ISCSGSSSNI....GSNYVYWYQQLSGKAP..KLLIYNN
.......NQRPSGVP.DRFSGSK..SGTSASLAISGLQSEDEADYYCAAWDSSLSG
>NW_001095158|IGLV1-11*01|Macaca mulatta|F|V-REGION| |230 nt|1| | | |76 AA|76+18=94|partial in 3'| |
QSVLTQPPS.VSAAPGQRVT.ISCSGSSSNI....GRSYVSWYQQVPGTAP..KLLIYQD
.......NKRPSGVS.DRFSGSK..SGTSASLAI
>NW_001095158|IGLV1-12*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSGDPGQRVT.ISCTGSSSNI....GGYDVYWYQQLPGTAP..KLLIYEN
.......NKRPSGVS.DRFSGSK..SGTSASLTITGLQSEDEAEYYCETWDNSLNG
>NW_001095158|IGLV1-13*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSGDPGQRVT.ISCTGSSSNI....GGYYVYWYQQFPGTAP..KLLIYDN
.......NKRPSGVS.DRFSGSK..SGTSASLTITGLQPGDEADYYCGAWDSSLSA
>NW_001095158|IGLV1-14*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSGAPGQRVT.ISCTGSSSNI....GGYYVSWYQQLPGTTP..KLLIYQD
.......NKRPSGVS.DRFSGSK..SGTSASLTITGLQTEDEADYYCLSYDSSLSA
>NW_001095158|IGLV1-15*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.ASEAARKSVT.ISCSGSSSNI....GSNSVSWYQQLPGTAP..KLLIYYN
.......DQRASGVS.DRFSGSK..SGTSASLAISGLQTEDEADYYCAAWDDSLSG
>NW_001095158|IGLV1-2*01|Macaca mulatta|P|V-REGION| |299 nt|1| | | |99 AA|99+17=116| | |
QSVLTQLPS.VPGAQGHRVT.VSCTGSNSNME...GGCNVNQYQ*LPGADL..RLLMDSD
.......RNWPSMVP.DQFSGSK..TGNSASLTITGL*VEDEAADHCQSFDSNAST
>AC215787|IGLV1-2*02|Macaca mulatta|P|V-REGION|43203..43501|299 nt|1| | | |99 AA|99+17=116| | |
QSVLTQLPS.VPGAQGHRVT.VSCTGSNSNME...GGCNVNQYQ*LPGADL..RLLMDSD
.......RNWPSMVP.DQFSGSK..AGNSASLTVTGL*VEDEADDHCQSFDSNAST
>NW_001095158|IGLV1-6*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSAAPGQRVT.ISCSGSSFNF....RRYYVSWYQQLPGAAP..KLLIYDV
.......NKRPSGVS.DRFSGSQ..SGTSATLGISGLRPEDEADYYCSAWDSSLST
>AC214063|IGLV1-6*02|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| |rev-compl|
QSVLTQPPS.VSAAPGQRVT.ISCSGSSFNF....RRYYVSWYQQLPGTAP..KLLIYDV
.......NKQPSGVS.DRFSGSQ..SGTSATLGISGLRPEDEADYYCSAWDNSLSA
>NW_001095158|IGLV1-7*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSGAPGQRVT.ISCTGSSSNI....GGYYVQWYQQLPGTAP..KLLIYEN
.......NKRPSGVS.DRFSGSQ..SGTSASLTITGLQSEDEADYYCQSYDSSLSA
>NW_001095158|IGLV1-8*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.ASGAPGQSVT.ISCSGSSSNI....GSNYVYWYQQLPGTAP..KLLIYYS
.......NQRPSGVP.DRFSGSK..SGTSASLAITGLRSEDEADYYCAAWDNSLSS
>NW_001095158|IGLV1-9*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.ASGAPGQSVT.ISCSGSSSNI....RGNGVHWYQQLSGMAP..KLLIYNN
.......NQRPSGVP.DRFSGSK..SGTSASLAITGLQSEDEADYYCEAWDNSLSG
>NW_001095158|IGLV10-1*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+18=116| | |
QAGLTQPPS.VSKGLRQTAT.LTCTGNSNNV....GNQGAAWLQQHQGHPP..KLLSYRN
.......NNRPSGIS.ERFSASR..SGNTASLTITGLQPEDEADYYCSAWDSSLSA
>NW_001095158|IGLV11-1*01|Macaca mulatta|F|V-REGION| |312 nt|1| | | |104 AA|104+11=115| | |
QPVLTQPPS.LSASPGASAR.LPCTLSSDLSV...GSKNMYWYQQKPGSAP..RLFLYYY
SD...SDKQLGPGVP.NRVSGSKETSSNTAFLLISGLQPEDEADYYCQVYDSSAN
>AM055970|IGLV1S1*01|Macaca mulatta|F|V-REGION|144..440|297 nt|1| | | |99 AA|99+17=116| | |
QSVLTQPPS.VSGAPGQRVT.ISCTGSSSNIG...AGYYVQWYQQLPGTAP..KLLIYEN
.......NKRPSGVS.DRFSGSK..SGTSASLTITGLQSEDEADYYCQSYDSSLSA
>AM055971|IGLV1S2*01|Macaca mulatta|F|V-REGION|144..440|297 nt|1| | | |99 AA|99+17=116| | |
QSVLTQPPS.VSGAPGQRVT.ISCTGSSSNIG...AGYGVQWYQQLPGTAP..KLLIYEN
.......NKRPSGVS.DRFSGSQ..SGTSASLTITGLQSEDEADYYCLSYDSSLSA
>AM055972|IGLV1S3*01|Macaca mulatta|F|V-REGION|144..437|294 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSGAPGQRVT.ISCTGSSSNI....GGYYVSWYQQLPGTTP..KLLIYQD
.......NKRPSGVS.DRFSGSK..SGTSASLTITGLQTEDEADYYCLSYDSSLSA
>AM055973|IGLV1S4*01|Macaca mulatta|F|V-REGION|150..443|294 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.ASGAPGQSVT.ISCSGSSSNI....GGNNVYWYQQLPGTAP..KLLIYYS
.......NQRPSGVP.DRFSGSK..SGTSASLAITGLRSEDEADYYCAAWDDSLSS
>AM056014|IGLV1S5*01|Macaca mulatta|P|V-REGION|137..433|297 nt|1| | | |99 AA|99+17=116| | |
QSVLTQLPS.VPGAQGHRVT.VSCTGSNSNME...GGCNVNQYQ*LPGADL..RLLMDSD
.......RNWPSMVP.DQFSGSK..AGNSASLTVTGL*VEDEAADHCQSFDSNAST
>AM056015|IGLV1S6*01|Macaca mulatta|F|V-REGION|143..436|294 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSAAPGQKVT.ISCSGSSSNI....GRSYVSWYQQVPGTAP..KLLIYDN
.......NKRPSGVS.DRFSGSK..SGTSASLAITGLQTGDEADYYCGAWDSSLSA
>AM056016|IGLV1S7*01|Macaca mulatta|F|V-REGION|143..436|294 nt|1| | | |98 AA|98+18=116| | |
QSVLTQPPS.VSAAPGQKVT.ISCSGSSSNI....GRSYVSWYQQVPGTAP..KLLIYQD
.......NKRPSGVS.DRFSGSK..SGTSASLAITGLQTGDEADYYCSAWDSSLSA
>NW_001095158|IGLV2-10*01|Macaca mulatta|F|V-REGION| |129 nt|1| | | |43 AA|43+5=48|partial in 3'| |
QSAPIQSPS.VSGSLGQSVT.ISCTGNSSDIG...RYNYVSWYRQQPG
>NW_001095158|IGLV2-3*01|Macaca mulatta|F|V-REGION| |297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPRS.VSGSPGQSVT.ISCTGTSSDIG...GYNYVSWYQQHPGTAP..KLMIYAV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCCSYAGSYTF
>NW_001095158|IGLV2-5*01|Macaca mulatta|ORF|V-REGION| |297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPPS.VSKSLGQSVT.ISCAGTSSGIA...SYSDVSWYQQHPGTAP..RLLIYRV
.......SNRPSGVS.DRFSGFK..SGSTASLTISGLQAEDEAIYYCCSYRSGSTF
>NW_001095158|IGLV2-7*01|Macaca mulatta|F|V-REGION| |297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPPS.MSGSPGQSVT.ISCTGTSSDIG...GYNRVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSNTF
>NW_001095158|IGLV2-8*01|Macaca mulatta|F|V-REGION| |297 nt|1| | | |99 AA|99+17=116| | |
QAAPTQSPS.VSGSAGQSVT.ISCTGTSSDIG...YYNAVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSGTF
>NW_001095158|IGLV2-9*01|Macaca mulatta|F|V-REGION|2569157..2569446|297 nt|1| | | |99 AA|99+17=116| | |
QAALTQSPS.VSGSPGQSVT.ISCTGTSSDIG...GYNRVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYASSSTF
>AM055976|IGLV2S1*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPPS.VSGSPGQSVT.ISCTGTSSDIG...GYNYVSWYQQHPGKAP..KLMIYDV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSNTF
>AM055985|IGLV2S10*01|Macaca mulatta|F|V-REGION|139..435|297 nt|1| | | |99 AA|99+17=116| | |
QSAPIQSPS.VSGSLGQSVT.ISCTGTSSDIG...RYNYVSWYRQQPGTTT..KLMMYKV
.......NMRPSGVS.DRFSGSK..SGNTASLTISGLQAEDKADYYCSSYEASDTF
>AM055986|IGLV2S11*01|Macaca mulatta|F|V-REGION|139..435|297 nt|1| | | |99 AA|99+17=116| | |
QSAPIQSPS.VSGSLGQSVT.ISCTGTSSDIG...RYNYVSWYRQQPGTTT..KLMMYKV
.......NMRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYEASDTF
>AM055977|IGLV2S2*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAAPTQPPS.VSGSPGQSVT.ISCTGTSSDIG...GYNYVSWYQQHPGKAP..KLMIYDV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSNTF
>AM055978|IGLV2S3*01|Macaca mulatta|F|V-REGION|149..445|297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPRS.VSGSPGQSVT.ISCTGTSSDIG...GYNYVSWYQQHPGTAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSNTF
>AM055979|IGLV2S4*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAAPTQSPS.VSGSPGQSVT.ISCTGTSSDIG...GYNRVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSGTF
>AM055980|IGLV2S5*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAAPTQSPS.VSGSPGQSVT.ISCTGTSSDIG...GYNRVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYASSSTF
>AM055981|IGLV2S6*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAAPTQPPS.VSGSPGQSVT.ISCTGTSSDIG...YYNAVSWYQQHPGKAP..KLMIYEV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCSSYAGSNTF
>AM055982|IGLV2S7*01|Macaca mulatta|F|V-REGION|147..443|297 nt|1| | | |99 AA|99+17=116| | |
QSAPTQPPS.VSGSPGQSVT.ISCTGTSSDVG...GYNYVSWYQQHPGKAP..KLMIYGV
.......SNRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCCSYTTSSTF
>AM055983|IGLV2S8*01|Macaca mulatta|P|V-REGION|147..443|297 nt|1| | | |99 AA|99+17=116| | |
QSAPTQPPS.VSGSPGQSVT.ISCTGTSSDVG...GYNYVSWYQQHPGKAP..KLMIYDV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCCSYTTSSTF
>AM055984|IGLV2S9*01|Macaca mulatta|F|V-REGION|151..447|297 nt|1| | | |99 AA|99+17=116| | |
QAALTQPPS.VSKSLGQSVT.ISCTGTSNDVG...GYNDVSWYQQHPGTAP..RLLIYDV
.......SKRPSGVS.DRFSGSK..SGNTASLTISGLQAEDEADYYCCSYRSGSTF
>NW_001095158|IGLV3-1*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SYELTQPPS.VSVSPGQTAR.ITCGGDNLG......SKYVHWYQQKPAQAP..VLVIYYD
.......SDRPSGIP.ERFSGSK..SGNTATLTISGVEAGDEADYYCQVWDSSSDH
>NW_001095158|IGLV3-10*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SYELTQPPS.VSVSPGQTAR.ITCSGDALP......KKYAYWFQQKPGQSP..VLIIYED
.......SKRPSGIP.ERFSGSS..SGTVATLTISGAQVEDEADYYCYSTDSSGNH
>NW_001095158|IGLV3-12*01|Macaca mulatta|F|V-REGION|2582228..2582511|47 nt|1| | | |15 AA|15+1=16|partial in 3'| |
SYELTQPLS.VSVALG
>NW_001095158|IGLV3-19*01|Macaca mulatta|F|V-REGION| |299 nt|1| | | |99 AA|99+20=119| | |
SYELTQSPS.VSVSPGQPAS.ITCSGDKFG......NAYAYWYQQKPGQVP..VLVIYKN
.......SNRPSGIP.ERFSGSN..SGNTATLTISGVEAGDEADYYCATSYGSGSSWQ*
>NW_001095158|IGLV3-2*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVR.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>NW_001095158|IGLV3-3*01|Macaca mulatta|F|V-REGION| |284 nt|1| | | |94 AA|94+20=114| | |
SYELTQPPL.VSVSPGQTAR.ITCSGDVLK......ENYADWYQQKPGQAP..VLLIHED
.......SKRPSGIP.ERFSGST..SGDTTTLTISSTLSEDEADYSCFSGNENN
>NW_001095158|IGLV3-4*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SYDVTQPRS.VSVSPGQTAR.ITCGGDNIG......SKVVHWYQQKPAQAP..VLVIYRD
.......SKRPSGIP.ERFSGSN..SGNTATLTISGVEAGDEADYYCQVWDSSSDH
>NW_001095158|IGLV3-5*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SSELTQDPA.VSVALGQTVR.ITCQGDSLR......SYYASWYQQKPGQAP..VLVVYGN
.......NNRPSGIP.ERFSGSS..SGNTASLTITGAQVEDEADYYCDSWDSSGTH
>NW_001095158|IGLV3-6*01|Macaca mulatta|F|V-REGION| |290 nt|1| | | |96 AA|96+20=116| | |
SYELTQPPS.VSAASGQTAR.ITCGGDNIG......SKNVHWYQQKPAQAP..VLVIYAD
.......SKRPSGIP.ERFSGSN..SGNTATLTISRVEAGDEADYYCQVWDSSSKY
>NW_001095158|IGLV3-9*01|Macaca mulatta|ORF|V-REGION| |285 nt|1| | | |95 AA|95+21=116| | |
SSELAQESA.VSVALLQMAR.ITCQGDGIG......NYYTNWCQQKPGRAL..VGVTCGN
.......N.CPSGIP.DKFSGSK..SEKKAILTITGTQVEDEADYYFLSRQQWYSS
>AM055987|IGLV3S1*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055997|IGLV3S10*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSGLTQEPA.LSVALGHTVR.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......TNRPSGIP.GRFSGSW..SGNTGSLTITGAQVEDEADYYCGSWDNSGNH
>AM055998|IGLV3S11*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSGLTQEPA.LSVALGHTVR.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......TNRPSGIP.GRFSGSW..SGNTGSLTITGAQVEDEADYYCGSWDNSGNH
>AM055999|IGLV3S12*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQKPA.LSVALGHTVR.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......TNRPSGIP.GRFSVSW..SGNTASLTITGAQVEDEADYYCGSWDNSGNH
>AM056000|IGLV3S13*01|Macaca mulatta|F|V-REGION|179..468|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQDPA.VSVALGQTVR.ITCQGDSLR......SYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITGAQVEDEADYYCGSWDNSGNH
>AM056001|IGLV3S14*01|Macaca mulatta|F|V-REGION|176..459|282 nt|1| | | |94 AA|94+20=114| | |
SYELTQPPS.VSVSPGQTAK.ITCSGEILA......KKYARWFQQKPGQAP..VLVIYKD
.......SERPSGIP.ERFSGSS..SGTTVTLTVSGAQAEDEADYYCYSGDDNN
>AM056002|IGLV3S15*01|Macaca mulatta|F|V-REGION|181..464|282 nt|1| | | |94 AA|94+20=114| | |
SYELTQPPS.VSVSPGQTAK.ITCSGEILA......KKYARWFQQKPGQAP..VLVIYKD
.......SERPSGIP.ERFSGSS..SGTTVTLTVSGAQAEDEADYYCYSGDDNN
>AM056003|IGLV3S16*01|Macaca mulatta|F|V-REGION|176..459|282 nt|1| | | |94 AA|94+20=114| | |
SYELTQPPS.VSVSLGQTAK.ITCSGDVLA......KYYAHWFQQKPGQAP..VLVIYKD
.......SERPSGIP.ERFSGSS..SGTTVTLTISGAQAEDEADYYCYSGDDNN
>AM056018|IGLV3S18*01|Macaca mulatta|ORF|V-REGION|180..467|288 nt|1| | ||97 AA|97+19=116| | |
SSELAQESA.VSVALLQMARRITCQGDGIG......NNYTNWCQQKPGRAL.XVGVTCG.
.......NNCPSGIP.DKFSGSK..SEKKAILTITGTQVEDEANYYFLSRQQWYSS
>AM056019|IGLV3S19*01|Macaca mulatta|P|V-REGION|178..470|291 nt|1| | | |97 AA|97+19=116| | |
SSQLTQEPS.VSVALGQMAR.ITCQGDGIG......TYYANWYEQKPRYQK..PCADHLW
......*QQPALRIP.D*FSGSK..SGNTATLIISRTQTEDKADCY*QSPDSSGTH
>AM055988|IGLV3S2*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>AM056020|IGLV3S20*01|Macaca mulatta|P|V-REGION|178..470|291 nt|1| | | |97 AA|97+19=116| | |
SSQLTQEPS.VSVALGQMAR.ITCQGDGIG......TYYANWYEQKPRYQK..PCADHLW
......*QQPALRIP.D*FSGSK..SGNMATLTISGTQTEDKADCY*QSPDSSGTH
>AM055989|IGLV3S3*01|Macaca mulatta|F|V-REGION|177..466|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIS.ERFSGSS..SGNTASLTITGAQVEDEADYYCDSWDSSGTH
>AM055991|IGLV3S4*01|Macaca mulatta|F|V-REGION|177..466|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNRGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055992|IGLV3S5*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055993|IGLV3S6*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055994|IGLV3S7*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSGLTQEPA.LSVALGHTVR.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......TNRPSGIP.GRFSGSW..SGNRGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055995|IGLV3S8*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSELTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNTGSLTITAAQVEDEADYYCNSWDSSGTH
>AM055996|IGLV3S9*01|Macaca mulatta|F|V-REGION|178..467|288 nt|1| | | |96 AA|96+20=116| | |
SSGLTQEPA.LSVALGHTVS.MTCQGDSLK......TYYASWYQQKPGQVP..VLVIYGN
.......NYRPSGIP.GRFSVSW..SGNRGSLTITAAQVEDEADYYCGSWDNSGNH
>NW_001095158|IGLV4-2*01|Macaca mulatta|F|V-REGION| |299 nt|1| | | |99 AA|99+15=114| | |
QPVLTQSPS.ASASLGASVK.LTCTLSSGHS.....SYAIAWHQQQQGKAP..RYLMRLN
SD...GSHSKGDGIP.DRFSGSS..SGAERYLTISNLQSEDEADYYCQTWDTGI
>NW_001095158|IGLV4-3*01|Macaca mulatta|ORF|V-REGION| |320 nt|1| | ||106 AA|106+13=119| | |
LSVLTQPPS.ASALLSASAK.LTCTLSSEHS.....TYTTERYQQRPGRSPSSQCITKVK
SD...GSHSKGNRIP.NHLLGSS..SGAECYLIIFNLQSQDEAKYHCGDSHMFDGEDG*
>NW_001095158|IGLV4-4*01|Macaca mulatta|F|V-REGION| |314 nt|1| | | |104 AA|104+15=119| | |
LPVLTQPPS.ASASLGASVK.LTCTLSSEHS.....NYFIFWYQQRPGRSP..RYIMKVN
SD...GTQSKGDGIP.DRFLGSS..SGADRYLTISNLQSDDEAEYYCGECHTIDGQYG*
>AM056004|IGLV4S1*01|Macaca mulatta|F|V-REGION|161..459|297 nt|1| | | |99 AA|99+15=114| | |
QPVLTQSPS.ASASLGASIK.LTCTLSSGHS.....SYTIAWHQQQQGKAP..RYLMWLK
SD...GSHSKGDGIP.DRFSGSS..SGAERYLTISNLQSEDEADYYCQTWDTGI
>AM056005|IGLV4S2*01|Macaca mulatta|F|V-REGION|161..459|297 nt|1| | | |99 AA|99+15=114| | |
QPVLTQSPS.ASASLGASVK.LTCTLSSGHS.....SYTIAWHQQQQGKAP..RYLMWLK
SD...GSHSKGDGIP.DRFSGSS..SGAERYLTISNLQSEDEADYYCQTWDTGI
>AM056006|IGLV4S3*01|Macaca mulatta|F|V-REGION|161..459|297 nt|1| | | |99 AA|99+15=114| | |
QPVLTQSPS.ASASLGASVK.LTCTLSSGHS.....SYTIAWHQQQQGKAP..RYLMWLK
SD...GSHSKGDGIP.DRFSGSS..SGAERYLTISNLQSEDEADYYCQTWDTGI
>AM056007|IGLV4S4*01|Macaca mulatta|F|V-REGION|161..459|297 nt|1| | | |99 AA|99+15=114| | |
QPVLTQSPS.ASASLGASVK.LTCTLSSGHS.....SYAIAWHQQQQGKAP..RYLMRLN
SV...GSHSKGDGIP.DRFSGSS..SGAERYLTISNLQSEDEADYYCQTWTTGI
>NW_001095158|IGLV5-1*01|Macaca mulatta|P|V-REGION| |317 nt|1| | | |105 AA|105+11=116| | |
QSCADSVTL.ALESPGASAR.LTCTLSSDINV...NAKSISWYQQKPGNHF..QYLLSYY
SD...SVKHQGYGVP.GRFSGSKDASTN*GILHISGLQPEDEADHHCKIRHDSING
>NW_001095158|IGLV5-10*01|Macaca mulatta|F|V-REGION| |312 nt|1| | ||104 AA|104+11=115| | |
QPVLTQPAS.LSASPGASAS.LTCTFSGGTNV...GDYTIHWYQQKPGSPP..RYLLKYK
SD...SDKHQGSGVP.SRFSGSKDASANTGILRISGLQSEDEADYYCAIGRSSAS
>AC214063|IGLV5-10*02|Macaca mulatta|F|V-REGION| |312 nt|1| | ||104 AA|104+11=115| | |
QPVLTQPAS.VSASPGASAS.LTCTFSGGTNV...GDYTIHWYQQKPGSPP..RYLLKYK
SD...SDKHQGSGVP.SRFSGSKDASANTGILRISGLQSEDEADYYCAIGRSSAS
>NW_001095158|IGLV5-11*01|Macaca mulatta|F|V-REGION| |312 nt|1| | | |104 AA|104+11=115| | |
QPVLTQPTS.LSASPGASAR.LTCTLRSGISV...GSYRIFWYQQKPGSPP..RYLLNYH
TD...SDKHQGSGVP.SRFSGSKDASANAGILLISGLQSEDEADYYCMIWHNNAS
>NW_001095158|IGLV5-13*01|Macaca mulatta|F|V-REGION| |312 nt|1| | | |104 AA|104+11=115| | |
QPVLTQPTS.LSASPGASVR.LTCTLRSGISV...GGYNIHWYQQKPGSPP..RYLLYYY
SD...SNKGQGSGVP.SRFSGSKDASANAGILLISGFQSEDEADYYCTTWHNNAS
>NW_001095158|IGLV5-14*01|Macaca mulatta|F|V-REGION| |312 nt|1| | | |104 AA|104+11=115| | |
KPMLTQPAS.LSASPGASAS.LTCTFSGGINV...AGYHIFWYQQKPGSPP..RYLLRYK
SD...SDKGQGSGVP.SRFSGSKDASANTGILRISGLQSEDEADYYCAIGHSSGP
>NW_001095158|IGLV5-15*01|Macaca mulatta|P|V-REGION|2179123..2179431|309 nt|1| | ||103 AA|103+12=115| | |
QPVLTQPPS.FSTS.GASAS.FTCTLRSDINV...SRYRVYWYQQKPESPP..RYLLSYY
SD...SYKHQGSGVP.SRFSGSKDASVNAGLLLISGLQPEDEADYYCKTGYGNAS
>AM056013|IGLV5-15*02|Macaca mulatta|[P]|V-REGION|162..470|309 nt|1| | ||103 AA|103+12=115| | |
QPVLTQPPS.FSTS.GASAS.LTCTLRSDINV...SRYRIYWYQQKPESPP..RYLLSYY
SD...SYKHQGSGVP.SRFSGSKDASVNAGLLLISGLQPEDEADYYCKTGYGNAS
>NW_001095158|IGLV5-5*01|Macaca mulatta|F|V-REGION| |312 nt|1| | | |104 AA|104+11=115| | |
QPVLTQPPS.LSASQGASAR.LSCTLSSGFSA...DLYWIYWYQHKPGSPP..RYLLSLY
QN...SLHDLGSGVP.RRISGLMEDWSNKGLLLISDLQPEDEADYYCMIEHGRAS
>NW_001095158|IGLV5-6*01|Macaca mulatta|F|V-REGION| |317 nt|1| | | |105 AA|105+11=116| | |
QSVLTQPPS.LSASLEALAR.LTCTLSSGISV...GGKIVYWYQQKPGSNP..RYLLSYY
SE...SSKHQGSGVP.GRFSGSKDASTNSGILHVSGLQPEDEADYYCKIWHDSINA
>NW_001095158|IGLV5-7*01|Macaca mulatta|F|V-REGION| |317 nt|1| | | |105 AA|105+11=116| | |
QPVLTQPSS.HSASPGAPAR.LTCTLSSGFSV...GDFWIRWYQQKPGSPP..RYLLYYH
SD...SDKHQGSGVP.SRFSGSNDASANAGILHISGLQPEDEADYYCCTWHGNSKT
>AC214063|IGLV5-8*01|Macaca mulatta|P|V-REGION| |315 nt|1| | | |105 AA|105+11=116| |rev-compl|
QPVLTQPLS.LSASLGARAT.LPCTLGSGLKV...EYYLIWWYQHEPGSTP..RHLLNFK
*D...SDKHQGSGVP.TCSSGSNSALSSAGIPHITGLQSEDKVDYYRTT*HDSCKT
>AM056011|IGLV5S2*01|Macaca mulatta|F|V-REGION|157..468|312 nt|1| | | |104 AA|104+11=115| | |
KPMLTQPAS.LSASPGASAS.LTCTFSGGINV...AGYHIFWYQQKPGSPP..RYLLRYK
SD...SDKGQGSGVP.SRFSGSKDASANAGILRISGLQSEDEADYYCAIGRSSAS
>AM056012|IGLV5S3*01|Macaca mulatta|F|V-REGION|156..467|312 nt|1| | | |104 AA|104+11=115| | |
QPVLTQPAS.LSASPGASAR.LTCTLSSGISV...GSYRIFWYQQKPGSPP..RYLLNYH
TD...SDKHQGSGVP.SRFSGSKDASANAGILLISGLQSEDEADYYCMIWHNNAS
>NW_001095158|IGLV6-1*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+16=114| | |
EVVFTQPHS.VSGSPGQTVT.ISCTRSSGSI....DSEYVQWYQQRPGSAP..TTVIYKD
.......NQRPSGVP.DRFSGSIDSSSNSASLAISGLKSEDEADYYCQSYDSYN
>NW_001095158|IGLV6-2*01|Macaca mulatta|F|V-REGION| |299 nt|1| | | |99 AA|99+16=115| | |
EVVFTQPHS.VSGSPGQTVT.ISCTRSSGSI....DSEYVQWYQQRPGNAP..TTVIYKD
.......NQRPSGVP.DRFSGSIDSSSNSASLAISGLKSEDEADYYCQSADGSYN
>NW_001095158|IGLV6-5*01|Macaca mulatta|F|V-REGION| |299 nt|1| | | |99 AA|99+16=115| | |
EVVFTQPHS.VSGSPGQTVT.ISCTHSSGSI....DNSYVYWYQQRPGSAP..TTVIYND
.......DQRPSGVP.DRSSGSIDSSSNSASLTISGLKSEDEADYYCQSYDSSGY
>NW_001095158|IGLV7-1*01|Macaca mulatta|F|V-REGION| |294 nt|1| | | |98 AA|98+17=115| | |
QAVVTQEPS.LTVSPGGTVT.LTCGSSAGAVT...GSHYPYWFQQKPGQAP..RTLIYDT
.......SNKLSWTP.ARFSGSL..LGGKAALTLSGAQPEDEAEYYCWLHYSGAQ
>NW_001095158|IGLV7-2*01|Macaca mulatta|F|V-REGION| |294 nt|1| | | |98 AA|98+17=115| | |
QAVVTQEPS.LTVSPGGTVT.LTCASSTGAVT...SGHYPHWFQQKPGQAP..KTLIYDT
.......SNKLSWTP.ARFSGSL..AGGKAALTLSGAQPEDEAEYYCWLYYSGAQ
>NW_001095158|IGLV7-3*01|Macaca mulatta|F|V-REGION| |294 nt|1| | | |98 AA|98+17=115| | |
QAVVTQEPS.MTVSPGGTVT.LTCASSTGAVT...SGHSPHWFQQKPGQAP..KTLIYNT
.......NYKHSWTP.ARFSGSL..LGGKAALTLSGAQPEDKAEYYCWLYYSGAQ
>NW_001095158|IGLV7-4*01|Macaca mulatta|F|V-REGION| |294 nt|1| | | |98 AA|98+17=115| | |
QAVVTQEPS.LTVSPGGTVT.LTCASSTGAVT...SGHSPHWCQQKPGQAP..RTLIYNT
.......SFKHSWTP.ARFSGSL..LGGKAALILSGAQPEDEAEYYCLLHYSGAQ
>NW_001095158|IGLV7-5*01|Macaca mulatta|P|V-REGION| |294 nt|1| | | |98 AA|98+17=115| | |
QAVVT*EPS.LTVFPGGTVT.LVCASSAGAVT...GGHYPYWVHQKPGQAP..RTLIYHA
.......SNKHSWIP.AQFSGSF..LGGKAALTLSGVQPKDEAEYYCLLHHSGAW
>NW_001095158|IGLV8-1*01|Macaca mulatta|F|V-REGION| |296 nt|1| | | |98 AA|98+17=115| | |
ETVVTQEPS.LSVYPGGTVT.LTCGLSSGSVS...TSNYPSWYQQTPGQAP..RTLIYST
.......NTRPSGVP.DRFSGSI..LGNKAALTITGAQADDESDYYCTLYMGSGI
>NW_001095158|IGLV9-1*01|Macaca mulatta|F|V-REGION| |317 nt|1| | | |105 AA|105+14=119| | |
QPVLTQPSS.ASASLGASVT.LTCTLSSGYS.....NYAVDWHQQRPGKGP..QFVMRVG
TG..GIVGSKGDGIP.DRFSGSG..SGLNRYLTIKNIQEEDESDYHCGADHGTGSSFV*

</pre>
<hr />





<a href="/genedb/doc">-> IMGT/GENE-DB Documentation</a><br />
<a href="/genedb/">-> IMGT/GENE-DB Query page</a><br />
<a href="/genedb/directlinks">-> IMGT/GENE-DB direct links</a><br />
<a href="/genedb/stats">-> IMGT/GENE-DB statistics</a><br />
<a href="/IMGTgenedbdoc/programversions.html">-> IMGT/GENE-DB program versions</a><br />
<a href="/IMGTgenedbdoc/dataupdates.html">-> IMGT/GENE-DB data updates</a>

<hr />

<p class="menu">
<a href="/">IMGT Home page</a>
| <a href="/IMGTrepertoire/">IMGT Repertoire (IG and TR)</a>
| <a href="/IMGTrepertoireMH/">IMGT Repertoire (MH)</a>
| <a href="/IMGTrepertoireRPI/">IMGT Repertoire (RPI)</a>
| <a href="/IMGTindex/">IMGT Index</a>
| <a href="/IMGTScientificChart/">IMGT Scientific chart</a>
| <a href="/IMGTeducation/">IMGT Education</a>
</p>



<span class="logosFoot">
<a href="https://www.cnrs.fr"><img src="/images/cnrs.png" alt="logo CNRS" style="vertical-align:middle;" /></a>
<a href="https://www.umontpellier.fr/"><img src="/images/logos/universite_mtp.png" alt="Universit&#233; de Montpellier" style="width:60px;vertical-align:middle;" /></a>
<a href="https://ec.europa.eu/info/research-and-innovation_en"><img src="/images/UE.png" alt="logo UE" style="vertical-align:middle;" /></a>
</span>



<p>&#169;  Copyright 1995-2020 IMGT&reg;, the international ImMunoGeneTics information system&reg;
			| <a href="/about/termsofuse.php">Terms of use</a> | <a href="/about/aboutus.php">About us</a> | <a href="mailto:contact-imgt@igh.cnrs.fr">Contact us</a> | <a href="/about/CitingIMGT.php">Citing IMGT</a></p>

<!-- #echo var=$LAST_MODIFIED -->

</body>
</html>

</body>
</html>