
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<link rel="stylesheet" type="text/css" href="css/GENEDBQuery.css"/>
<link rel="stylesheet" type="text/css" href="css/IndividualResult.css"/>
<link type="text/css" href="css/GENEDBQuery_old.css" rel="stylesheet" />
<link type="text/css" href="css/styleFoot.css" rel="stylesheet" />
<title>IMGT/GENE-DB Consultation</TITLE>
</head>
<body id="GENE-DB">
<table style="width:100%;">
<tr>
<td align="left" nowrap valign="middle" width="70%">
<font size="+3" color="#000099"><b>THANK YOU</b><br/>
<b>for using <a href="/genedb/doc">IMGT/GENE-DB</a></b></font>
</td>
<td class="notshow" align=right valign=top width="30%"><a href="/"><img SRC="images/logo_IMGT2.png" ALT="WELCOME !" class="mainlogo"/></a>
</td>
</tr>
<tr>
<td class="logolegendleft">IMGT&reg;, the international ImMunoGeneTics information system&reg;</td>
<td class="logolegend">http://www.imgt.org</td>
</tr>
</table>
<hr/>
<img src="images/logoGene-DB.png" ALT="Gene-DB logo" class="mainlogo" />

<table class="citation">
<tr>
<td class="citation">
<b>Citing IMGT/GENE-DB</b>: Giudicelli, V. et al. Nucleic Acids Res., 33: D256 - D261 (2005).
PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/15608191/">15608191</a>
<a href="/PDF/NAR/33_D256-D261_2005.pdf"><img src="/images/pdficon.png" alt="pdf" /></a>
</td>
</tr>
</table>
<p class="programversion">IMGT/GENE-DB program version: <a href="/IMGTgenedbdoc/programversions.html">3.1.30</a> (18 September 2020)<br/>
<a href="/IMGTgenedbdoc/dataupdates.html">IMGT/GENE-DB data updates </a>
</p>

<h2 class="subtitle"><a href="/IMGTScientificChart/SequenceDescription/IMGTreferencesequences.html" >IMGT/GENE-DB reference sequences</a> in FASTA format:</h2>
<h2 class="subtitle">Amino acids  sequences with gaps according to the <a href="/IMGTScientificChart/Numbering/IMGTnumberingCR.html">IMGT unique numbering</a><br /> for <span class="casimir">F+ORF+in-frame P with IMGT Gaps</span> alleles including orphons</h2>
<hr>
<pre>
The FASTA header contains 15 fields separated by '|':

1. IMGT/LIGM-DB accession number(s)
2. IMGT gene and allele name
3. species
4. IMGT allele functionality
5. exon(s), region name(s), or extracted label(s)
6. start and end positions in the IMGT/LIGM-DB accession number(s)
7. number of nucleotides in the IMGT/LIGM-DB accession number(s)
8. codon start, or 'NR' (not relevant) for non coding labels
9. +n: number of nucleotides (nt) added in 5' compared to the corresponding label extracted from IMGT/LIGM-DB
10. +n or -n: number of nucleotides (nt) added or removed in 3' compared to the corresponding label extracted from IMGT/LIGM-DB
11. +n, -n, and/or nS: number of added, deleted, and/or substituted nucleotides to correct sequencing errors, or 'not corrected' if non corrected sequencing errors
12. number of amino acids (AA): this field indicates that the sequence is in amino acids
13. number of characters in the sequence: nt (or AA)+IMGT gaps=total
14. partial (if it is)
15. reverse complementary (if it is)
</pre>
<hr/>
<b>Number of results = 113</b><br />
<pre>
>IMGT000064|IGHV1-105*01|Macaca mulatta_AG07107|F|V-REGION|753595..753890|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKKPGASVKISCKASGYTF....TDYYLHWVRQAPGKGLEWMGRVDPE.
.DGEAIHAQKFQ.DRVTITRDTSTDTAYMELSSLRSEDTAVYYCAT
>IMGT000064|IGHV1-107*01|Macaca mulatta_AG07107|P|V-REGION|743669..743964|296 nt|1| | | |98 AA|98+8=106| | |
QAQLVQSGA.EVKKPLSSMKISFKASGYTF....TYNFIHWV*QAPGQGLEWMGWMNTG.
.NGNTKYAQKFQ.GRVTITRDTSMSTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-123*01|Macaca mulatta_AG07107|P|V-REGION|629940..630237|298 nt|1| | | |99 AA|99+8=107| | |
QVQQVQSGA.EVKQPGSSVKVSCKASGYTF....TSSVMQWVRQAPGQGLQ*IGRICPY.
.NGNTKYAQNFQ.GRVTMTTDTSTSTACMELSRLRSEDTVVYYCAKD
>IMGT000064|IGHV1-130*01|Macaca mulatta_AG07107|F|V-REGION|580938..581233|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EVKKPGSSVKVSCKASGYIF....TDYYMHWVRQAPGQGLEWMGEINPK.
.TGGTNYAQKFQ.GRVTTTRDTSTSTAYMELSSLRSEDTAVYYCER
>IMGT000064|IGHV1-142*01|Macaca mulatta_AG07107|F|V-REGION|487860..488155|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EVKKPGASVKLSCKASGYTF....SIYAISWVRQAPGQGLEWMGGIIPL.
.VGITNYAQKFQ.GRVTITADTSTSTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-147*01|Macaca mulatta_AG07107|F|V-REGION|460137..460432|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKKPGASVKVSCKVSGYTF....TELSMHWVRQAPGKGLEWMGGVDPV.
.YGEIIHAEKFQ.GRVTMTEDTSTDTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-147D*01|Macaca mulatta_AG07107|F|V-REGION|342699..342994|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKKPGASVKVSCKVSGYTF....TELSMHWVRQAPGKGLEWMGGVDPV.
.YGEIIHAEKFQ.GRVTMTEDTSTDTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-170*01|Macaca mulatta_AG07107|F|V-REGION|217338..217633|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EIKQPGASVKLSCKASGYTF....TSYYMHWVRQAPGQGLEWIGLISPY.
.NGNKGYAQNFQ.GRVTITTDTSTSTGYMELSSLRSEDTAVYYCTR
>IMGT000064|IGHV1-187*01|Macaca mulatta_AG07107|P|V-REGION|43226..43521|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EG*KPGSSVKVSCKASGYIF....TSSAM*WV*QAPGQGLEWIGVIIIG.
.NGNTGYAQKFQ.GRVTITRDTSMSTAYVELSSLRSEDMAVYYCAA
>IMGT000064|IGHV1-188*01|Macaca mulatta_AG07107|F|V-REGION|33617..33912|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EVKKPGASVKVSCKASGFTF....GSYAISWVRQAPGQGLEWMGVIIPL.
.VGVTNYAEKFQ.GRVTITADTSTSTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-190*01|Macaca mulatta_AG07107|F|V-REGION|14814..15109|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EVKKPGASVKLSCKASGYTF....TSYYINWVRQAPGQGLDWMGWINPS.
.NGNTGYAQKFQ.GRVTMTRDTSTSTAYMELNSLRSEDTAVYYCAR
>IMGT000064|IGHV1-64*01|Macaca mulatta_AG07107|F|V-REGION|1066896..1067191|296 nt|1| | | |98 AA|98+8=106| | |
EMQLVQSEA.EVKKPGASVKISCKASGYTF....TYRYLHWLRQTPGQGLEWMGWITPY.
.NGNTNYAQKFQ.DRATITRDRSMSTAYMELSSLRSEDTAVYYCAR
>IMGT000064|IGHV1-65*01|Macaca mulatta_AG07107|F|V-REGION|1062793..1063088|296 nt|1| | | |98 AA|98+8=106| | |
QEQLVQSGA.EVKKPGASVKVSCKASGYIF....TSYVISWLRQAPGQGFEWMGGIHPG.
.YGSTSYAQKFQ.GRVTITADMSTSTVYMELSSLRSEDMAVYYCAA
>IMGT000064|IGHV1-83*01|Macaca mulatta_AG07107|P|V-REGION|907492..907787|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EGKKPGSSVKVSCKASGYTF....TSSAM*WVRQAPGQGLEWIGVIIIG.
.NGNTGYAQMFQ.GRVTITRDTSMSTAYVELSSLRSEDKALYYCAA
>IMGT000064|IGHV2-143*01|Macaca mulatta_AG07107|F|V-REGION|478539..478839|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESGP.ALVKPTQTLTLTCTFSGFSLT..TSGMGVGWIRQPPGKALEWLALIYWD.
..DDKRYSTSLK.SRLTISKDTSKNQVVLTMTNMDPMDTATYYCARG
>IMGT000064|IGHV2-151*01|Macaca mulatta_AG07107|F|V-REGION|368783..369083|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESGP.ALVKPTQTLTLTCTFSGFSIS..TTGTGVSWIRQPPGKALEWLASIYWD.
..DDKYYSTSLK.SRLTISKDTSKNQVVLTMTNMDPVDTATYYCARG
>IMGT000064|IGHV2-164*01|Macaca mulatta_AG07107|F|V-REGION|251451..251751|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESGP.ALVKPTQTLTLTCTFSGFSLT..TSGMGVGWIRQPPGKALEWLALIYWD.
..DDKRYSTSLK.SRLTISKDTSKNQVVLTMTNMDPVDTATYYCARG
>NW_001121240|IGHV2-164*02|Macaca mulatta_17573|F|V-REGION|53863..54163|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESGP.ALVKPTQTLTLTCTFSGFSLS..TSGMGVGWIRQPSRKTLEWLAHIYWD.
..DDKRYSTSLK.SRLTISKDTSKNQVVLTMTNMDPMDTATYYCARR
>IMGT000064|IGHV2-7*01|Macaca mulatta_AG07107|ORF|V-REGION|1447097..1447397|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESDP.ALVKPTQTLTLTCTFSGFSLS..TSGMGVDGIRQPPGKALEWLARIDWD.
..DDKYYSPSLK.SRLTISKDTSKNQVVLTMTNMDPVDTATYYCARR
>IMGT000064|IGHV2-89*01|Macaca mulatta_AG07107|F|V-REGION|835712..836012|301 nt|1| | | |100 AA|100+7=107| | |
QVTLKESGP.ALVKPTQTLTLTCTFSGFSIS..TTGTGVGWIRQPPGKALEWLASIYWN.
..DSKYYSTSLK.SRLTISKDTSKNQVVLTMTNMDPVDTATYYCARV
>IMGT000064|IGHV3-10*01|Macaca mulatta_AG07107|P|V-REGION|1411654..1411946|293 nt|1| | | |97 AA|97+8=105| | |
DMQLVESLG.SLL*PGASVRLSLAASGFTF....CRYWMSWVHQAPGKGLE*VSFLSSS.
.GGSTSYADSVK.GPFTISRDDAKKSLYLQVNSLRAEDSAMYYCG
>IMGT000064|IGHV3-102*01|Macaca mulatta_AG07107|F|V-REGION|773416..773717|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGR.GLVQPGGSLRLSCAVSGFTF....SDHYMSWVRQAPGKGPEWVGFMRNKA
NGGRTEYAASGK.GRFTISRDDSKSIASLQMSSLKTEDTAVYYCAR
>IMGT000064|IGHV3-104*01|Macaca mulatta_AG07107|F|V-REGION|761724..762022|299 nt|1| | | |99 AA|99+7=106| | |
EVQLVESGG.GLVQPGGSLRLSCVASGFSF....SDHYMDWVRQAPGKGLEWVSSISSG.
SGSTTLYPDSVK.GRFTISRDNAKNTVYLQMNSLRAEDTAVYYCAR
>IMGT000064|IGHV3-109*01|Macaca mulatta_AG07107|F|V-REGION|731174..731469|296 nt|1| | | |98 AA|98+8=106| | |
EVQLAESGG.GLVQPGGSLRLSCAASGFTF....SGYEMHWVRQAPGKGLESVSVIGGD.
.SSYTHYADSVK.GRFTISRDNAKNSLSLQMNSLRAADTAVYYCAR
>IMGT000064|IGHV3-110*01|Macaca mulatta_AG07107|F|V-REGION|714557..714858|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRVSCAASGFTF....SDYYMQWVRQAPGKGPEWVGFIRNKA
NGGTAEYAASVK.GRFTISRDDSKSIASLQMNSLKTEDTAVYYCTR
>IMGT000064|IGHV3-111*01|Macaca mulatta_AG07107|P|V-REGION|707072..707367|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDHYMDWVRQAPGKGLEWVSSVSGS.
.SSNTYYPDSVK.GRFTISRDNAKNTLYLQMDSLRAEDTAVYYCAR
>IMGT000064|IGHV3-112*01|Macaca mulatta_AG07107|F|V-REGION|703439..703740|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SSSAMHWVRQASGKGLEWVGRIRSKS
NNYETGYAASVK.GRFTISRDDSKNTAYLQMNSLKTEDTAVYYCTT
>IMGT000064|IGHV3-113*01|Macaca mulatta_AG07107|F|V-REGION|695611..695906|296 nt|1| | | |98 AA|98+8=106| | |
EVQLAESGG.GLVQPGGSLRLSCAASGFTF....SSYWMYWVRQAPGKGLEWVSRISSD.
.GSSTSYADSVK.GRFTISRENAKNSLYLQMNSLRAEDTAVYYCAK
>IMGT000064|IGHV3-117*01|Macaca mulatta_AG07107|F|V-REGION|657993..658285|293 nt|1| | | |97 AA|97+9=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SDYYMDWVRQAPGKGLEWVSRIKWW.
..GSTYYADSVK.GRFTISRENAKNTLYLQMDSLRAEDTAVYYCAR
>IMGT000064|IGHV3-12*01|Macaca mulatta_AG07107|F|V-REGION|1387438..1387739|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SNYWMNWVRQAPGKGLEWVGFIKNKA
DGGTAAYAESVK.GRFTISRDDSKNTLYLQMNSLKTEDTAVYYCTR
>NW_001121239|IGHV3-12*02|Macaca mulatta_17573|F|V-REGION|247716..248017|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SNYWMSWVRQAPGKGLDWVGFIKNKA
DGGTAAYAESVK.GRFTISRDDSKNTLYLQMSSLNTEDTAVYYCTR
>IMGT000064|IGHV3-124*01|Macaca mulatta_AG07107|F|V-REGION|626134..626426|293 nt|1| | | |97 AA|97+9=106| | |
VEQLVESGG.GLVQPGASLRLSCAASEFTF....SSYDMHWVRQAPGKGLEWVSGISIG.
..GGTYYPDSVK.GRFTISRDNAKNSLYLQMNSLRAEDTAVYYCAR
>IMGT000064|IGHV3-126*01|Macaca mulatta_AG07107|F|V-REGION|604795..605090|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVKPGGSLRLSCAASGFTF....DDYAMSWVRQAPGKGLEWVSRISWD.
.GGSTYYADSVK.GRFTISRDNAKNTLYLQMDRLRAEDTALYYCSR
>IMGT000064|IGHV3-128*01|Macaca mulatta_AG07107|F|V-REGION|590621..590916|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SSYDMSWVRQAPGKGLEWVSYISYT.
.GKTIYYADSVK.GRFTISRDNAKNSLSLQMSSLRAEDTAVYYCTR
>IMGT000064|IGHV3-129*01|Macaca mulatta_AG07107|P|V-REGION|583845..584146|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLFQPGGSLRVSCAASGFTF....SDYYMQWVRQAPGKGPEWVDLIRNKA
NGGTTE*SASVK.GRCTISRDDSKSIASLQMSSLNTEDTAVYYCTR
>IMGT000064|IGHV3-137*01|Macaca mulatta_AG07107|P|V-REGION|516603..516900|298 nt|1| | | |99 AA|99+8=107| | |
EVQLIESIE.DLRQPGKFLRLSCVASGFAC....SSY*MSQVHQSPGKGLE*AIDVKYD.
.GSQIYHAGSVK.GRFSISNDNSKHLLYLQMNSLRAEDMAMYGCT*G
>IMGT000064|IGHV3-14*01|Macaca mulatta_AG07107|ORF|V-REGION|1376375..1376669|295 nt|1| | | |98 AA|98+9=107| | |
EVQLVESEG.NFVQPGGSLRLSCAVSGFTF....SSCDLDWIRHAPGKGLEWVSGISIA.
..GSTYYANSVN.SLFAISRGNAKNTLYLQMNSLIAEDTAIYYCVSE
>NW_001121239|IGHV3-14*02|Macaca mulatta_17573|ORF|V-REGION|259121..259415|295 nt|1| | | |98 AA|98+9=107| | |
VVQLVESEG.NFVQPGGSLRLSCAVSGFTF....SSCDLDWIRHAPGKGLEWVSGISIA.
..GSTYYANSVN.SLFAISRENAKNTLYLQMNSLIAEDTAIYYCVSE
>IMGT000064|IGHV3-144*01|Macaca mulatta_AG07107|F|V-REGION|473609..473910|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDYAMDWVRQAPGKGPEWVGFIRSKA
YGGTAEYAASVK.GRFTISRDDSKNTAYLQMSSLKTEDTAVYYCTR
>NW_001121240|IGHV3-144*02|Macaca mulatta_17573|F|V-REGION|192..493|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDYAMDWVRQAPGKGLEWVGFIRSKA
YGGTAEYAASVK.GRFTISRDDSKNTAYLQMSSLKTEDTAVYYCTR
>IMGT000064|IGHV3-144D*01|Macaca mulatta_AG07107|F|V-REGION|363863..364164|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDYAMDWVRQAPGKGLEWVGFIRSKA
YGGTAEYAASVK.GRFTISRDDSKNTAYLQMSSLKTEDTAVYYCTR
>IMGT000064|IGHV3-152*01|Macaca mulatta_AG07107|P|V-REGION|320798..321089|292 nt|1| | ||97 AA|97+7=104| | |
EVHLVESGG.GLAKPEGSLRLSYAASAGFTF...SSYYMNCVRQAPGNRLEWVRQVNPN.
.GGSTYLTDSGK.GRFTIPRDNAKNTLHLQMNRLKTENTAVYYC
>IMGT000064|IGHV3-165*01|Macaca mulatta_AG07107|F|V-REGION|244673..244962|290 nt|1| | | |96 AA|96+10=106| | |
EVQLAESGG.GLVQPGGSLRLSCAASGFTI....SSYWMSWVCQAPGKGLEWLSDIYG..
..STMYYGDSVK.GLFTVSRDNAKNSLYLQMNSLRAEDTAVYYCTR
>NW_001121240|IGHV3-165*02|Macaca mulatta_17573|F|V-REGION|61020..61309|290 nt|1| | | |96 AA|96+10=106| | |
EVQLAESGG.GLVQPGGSLRLSCAASGFTI....SSYWMSWVCQAPGKGLEWLSDIYG..
..STMYYGDSVK.GLFTVSRDNAKNSLSLQMNSLRAEDTAVYYCTR
>IMGT000064|IGHV3-166*01|Macaca mulatta_AG07107|F|V-REGION|235627..235925|299 nt|1| | | |99 AA|99+7=106| | |
EVQLVESGGGLVQPGGGSLRLSCAASGFTF....SDDYMEWVRQAPGKGLEWVGQINPN.
.GGTTFLMDSVK.GRFTISRDNAKNTLYLQINSLKIEDTAVYYCTR
>NW_001121240|IGHV3-166*02|Macaca mulatta_17573|F|V-REGION|69832..70127|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDDYMEWVRQAPGKGLEWVGQINPN.
.GGTTFLMDSVK.GRFTISRDNAKNTLYLQINSLKIEDTAVYYCTR
>IMGT000064|IGHV3-168*01|Macaca mulatta_AG07107|F|V-REGION|223991..224286|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SDYYMDWVRQAPGKGLEWVSRISNG.
.GGSTWYADSVK.GRFTISRENAKNTLYLQMNSLRAEDTAVYYCAR
>NW_001121240|IGHV3-168*02|Macaca mulatta_17573|F|V-REGION|81515..81810|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SDYYMDWVRQAPGKGLEWVSRISNG.
.GGSTWYADSVK.GRFTISRENAKNTLYLQMNSLRAEDTAVYYCAK
>IMGT000064|IGHV3-173*01|Macaca mulatta_AG07107|F|V-REGION|164603..164898|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....GDYGMHWVRQAPGKGLEWVSSISNT.
.GKTVYYADSVK.GRFTISRDNAKNSLSLQMSSLRAEDTAVYYCTR
>NW_001121240|IGHV3-173*02|Macaca mulatta_17573|F|V-REGION|140440..140735|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....GDYGMHWVRQAPGKGLEWVSSISNT.
.GKTVYYADSVK.GRFTVSRDNAKNSLSLQMSSLRAEDTAVYYCTR
>IMGT000064|IGHV3-174*01|Macaca mulatta_AG07107|F|V-REGION|144162..144463|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDYYMYWVRQAPGKGLEWVGFIRSKA
YGGTAEYAASVK.GRFTISRDDSKSIAYLQMSSLKTEDTAVYYCTR
>NW_001121240|IGHV3-174*02|Macaca mulatta_17573|F|V-REGION|160994..161295|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SDYYMYWVRQAPGKGLEWVGFIRSKA
YGGTAEYAASVK.GRFTISRDDSKSIAYLQMSSLKTEDTAVYYCTR
>IMGT000064|IGHV3-176*01|Macaca mulatta_AG07107|ORF|V-REGION|134232..134533|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRPSCAASGFTF....SSSAMHRVRQASGKGLEWVGRIRSKS
NNYATEYAASVK.GRFTISRDDSKNTAYLQMNSLKTEDTAVYYCAR
>NW_001121240|IGHV3-176*02|Macaca mulatta_17573|F|V-REGION|170925..171226|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRPSCAASGFTF....SSSAMHWVRQASGKGLEWVGRIRSKS
NNYATEYAASVK.GRFTISRDDSKNTAYLQMNSLKTEDTAVYYCAR
>IMGT000064|IGHV3-191*01|Macaca mulatta_AG07107|F|V-REGION|3205..3502|298 nt|1| | | |99 AA|99+8=107| | |
EVQLVESGG.GVVQPGGSLRLSCAASGFTF....DDYAMHWVRQAPGKGLEWVSGISWS.
.GGSTYYADSVK.GRFTISRDNAKNSLYLQMGSLRAEDTALYYCAKD
>IMGT000064|IGHV3-24*01|Macaca mulatta_AG07107|F|V-REGION|1318019..1318314|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SSYWMHWVRQAPGKGLEWISAINSA.
.GSSTYYADSVK.GRFTISRENAKNTLYLQMDSLRAEDTALYYCAG
>NW_001121239|IGHV3-24*02|Macaca mulatta_17573|F|V-REGION|314591..314886|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SNYWMHWVRQAPGKGLEWISAINSA.
.GSSTYYADSVK.GRFTISRENAKNTLYLQMDGLRAEDTAVYYCAG
>IMGT000064|IGHV3-26*01|Macaca mulatta_AG07107|F|V-REGION|1298609..1298910|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SNVWMNWVRQAPGKGLEWVARIKRKA
DGETADYAASVK.GRFTISRDDSKNTLYLQMNSLKTEDTAVYYCTT
>NW_001121239|IGHV3-26*02|Macaca mulatta_17573|F|V-REGION|335828..336129|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGA.GLVQPGGSLRLSCAASGFTF....SNSWMSWVRQAPGKGLEWVARIKRKA
DGETADYAASVK.GRFTISRDDSKNTLYLQMNSLKTEDTAVYYCTT
>IMGT000064|IGHV3-28*01|Macaca mulatta_AG07107|F|V-REGION|1289028..1289320|293 nt|1| | | |97 AA|97+9=106| | |
EVQLVESGG.GLVQPGGSLRLSCAAPGFTS....GNSDLIWIRQAPGKGLEWVSYISSG.
..GSIYYSDSVK.GRFTISRDNAKNTLYLQMSSLRVEDTAVYYCAK
>IMGT000064|IGHV3-3*01|Macaca mulatta_AG07107|P|V-REGION|1484248..1484543|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAAFGFTF....SSY*MHWVHQAPGKWLKWVSAISNG.
.GINTYYPDSMK.GQFTISRDTTKNSLYLQMNSLRVEDTAVYYCMR
>IMGT000064|IGHV3-30*01|Macaca mulatta_AG07107|F|V-REGION|1276599..1276894|296 nt|1| | | |98 AA|98+8=106| | |
EVKLVESGG.GLVQPGGSLRLSCAASGFTF....SNYWMFWVRQAPGKGLEWVSSISGS.
.SSSTYYPDSVK.GRFTISRDNAKNTLYLQMNSLRAEDTAVHYCAR
>IMGT000064|IGHV3-31*01|Macaca mulatta_AG07107|F|V-REGION|1259949..1260241|293 nt|1| | | |97 AA|97+9=106| | |
EVQLVEYGG.GLVQPGGSLRLSC...GFTF....SVHFMSWVRQAPGKGPEWVGFMRNKA
NGGTAEYATSVK.GRFTISRDDSKSIAYLQMSSLNTEDTAVYYCAR
>IMGT000064|IGHV3-33*01|Macaca mulatta_AG07107|F|V-REGION|1250596..1250891|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCVASGFTF....SDYCMDWVRQASGKGLEWVSSISGS.
.SSNTYYPDSVK.GRFTISRDNAKNTLYLQMNSLRAEDTAVYYCAR
>IMGT000064|IGHV3-34*01|Macaca mulatta_AG07107|F|V-REGION|1234755..1235056|302 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLAQPGGSLRLSCAASGFTF....SDHYMDWVRQAPGKGLEWVGRIRNKA
NSYTTEYAASVK.GRFTISRDDSKNTLYLQMSSLKTEDTAVYYCAR
>IMGT000064|IGHV3-42*01|Macaca mulatta_AG07107|P|V-REGION|1197791..1198086|296 nt|1| | | |98 AA|98+8=106| | |
EAQLVESGG.GLVQPGGSLRLSCAASGFTF....SSTRMNWIRQSPGKRLEWVADIK*D.
.GSEKYYVDSVK.GRFTISRDNTKNSLYLQMNSLRAEDTAVYYCVR
>IMGT000064|IGHV3-44*01|Macaca mulatta_AG07107|P|V-REGION|1190943..1191240|298 nt|1| | | |99 AA|99+8=107| | |
EVQLIESTE.DLKQPGKLLRLSCVASRFAC....SSL*ISRIHQCPGKGLQ*VIGIKYD.
.GSQICHGDYVK.GRFSISKDNAKNSLYLQMNSLRAEDMALYGCT*G
>IMGT000064|IGHV3-45*01|Macaca mulatta_AG07107|P|V-REGION|1188159..1188452|294 nt|1| | | |98 AA|98+8=106| | |
EVQLLESGG.DLEQPGGSLRLSCTVSGFTF....SSNWMYWVCQAPGKGLEWVSCVSSD.
.WSSTRYIHSVK.F*FTIPRNNAKNTLYLEMNSLKAEDTTMYYCAR
>IMGT000064|IGHV3-49*01|Macaca mulatta_AG07107|F|V-REGION|1156429..1156724|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SSYGMSWVRQTPGKGLEWVAVIWYD.
.GSKKYYADSVK.DRFTISRDNSKNMLYLQMNNLKLEDTAVYYCGR
>IMGT000064|IGHV3-53*01|Macaca mulatta_AG07107|F|V-REGION|1135853..1136154|302 nt|1| | | |100 AA|100+6=106| | |
EAQLMETGG.GLVQPGGSLRLSCAASGFTF....SDHYMQWVRQAPGKGLEWVGLIRNKA
DGETTDYALSVK.GRFTISRDDSKSITYLQMNNLKTEDTAVYYCAR
>IMGT000064|IGHV3-54*01|Macaca mulatta_AG07107|F|V-REGION|1128066..1128361|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SDYYMHWVRQASGKGLEWVSRISNG.
.GGSTWYADSVK.GRFTISRENAKNTLYLQMDSLRAEDTAVYYCAR
>IMGT000064|IGHV3-6*01|Macaca mulatta_AG07107|F|V-REGION|1471210..1471507|298 nt|1| | | |99 AA|99+8=107| | |
EVQLVESGG.GLVQPGGSLRLSCTGSGFTF....SSYYMYWVRQAPGKGLEWVSAINTG.
.GGSTWYTDSVK.GRFTISKENAKNTLYLQMDSLRAEDTAVYYCAKD
>IMGT000064|IGHV3-62*01|Macaca mulatta_AG07107|P|V-REGION|1084763..1085058|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GVVQPGGSLRLSCAASGFTF....DDYAMGWVRQAPGKGLEWVSAISWD.
.GGSTGYADSVK.GRFTISRDNAKNSLYLQMGSLRAEDTALYYCAR
>IMGT000064|IGHV3-66*01|Macaca mulatta_AG07107|F|V-REGION|1055553..1055845|293 nt|1| | | |97 AA|97+9=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SSYAMQWVHQAPGKGLEWVSAIGPG.
..GDTYYADAVK.GRFTISRDNAKNSLYLQMNSLRAEDTAVYYCAR
>IMGT000064|IGHV3-72*01|Macaca mulatta_AG07107|F|V-REGION|997537..997832|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GVVQPGGSLRLSCAASGFTF....DDYAMGWVRQAPGKGLEWVSAISWN.
.GDSTYYADSVK.GRFTISRENAKNSLYLQINRLRAEDTALYYCAR
>IMGT000064|IGHV3-8*01|Macaca mulatta_AG07107|F|V-REGION|1432197..1432486|290 nt|1| | | |96 AA|96+10=106| | |
EVQLVESGG.GLVQPGRSLRPSCAASGFTF....SSYGMHWVRQAPEEGLVWVSYIGS..
..STMYYADSVK.GRFTISRDNAKNSLYLQMNSLRAEDTAVYYCVR
>NW_001121239|IGHV3-8*02|Macaca mulatta_17573|F|V-REGION|201644..201933|290 nt|1| | | |96 AA|96+10=106| | |
EVQLVESGG.GLVQPGRSLRPSCAASGFTF....SSYGMHWVRQAPEEGLVWVSYIGS..
..STMYYADSVK.GRFTISRDNAKNSLYLQMNSLRAEDTAVYYCVR
>IMGT000064|IGHV3-9*01|Macaca mulatta_AG07107|F|V-REGION|1422114..1422413|300 nt|1| | | |100 AA|100+6=106| | |
EVQLVESGG.GLVQPGGSLRLSCAASGFTF....SNYYMHWVRQAQGKGLEWVGLIRNKA
NSYTTEYAAAVK.GRFTISRDDSKNTLYLQMSSLKTEDTALYYCTK
>IMGT000064|IGHV3-94*01|Macaca mulatta_AG07107|F|V-REGION|815505..815800|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVESGG.GLVKPGGSLRLSCVASGFTF....SSYVMHWVRQAPGKGLEWVSVISES.
.GGTTYYADSVK.GRFTISRDNAKNSLFLQMNSLRAEDTAVYYCTR
>IMGT000064|IGHV3-96*01|Macaca mulatta_AG07107|P|V-REGION|806957..807254|298 nt|1| | | |99 AA|99+8=107| | |
EVQLTESTE.DLRQPGKFLRLSYVASRFAC....SSF*ISRVHQSPGKGLE*VIGIKYD.
.GSHIHHADSLK.GRFTISKDNAKNSLYLQMNSQIAEDIAM*GCT*G
>IMGT000064|IGHV3-97*01|Macaca mulatta_AG07107|F|V-REGION|803357..803649|293 nt|1| | | |97 AA|97+9=106| | |
EVQLVESGG.GLAKPGGSLRLSCAASGFTF....SSYAMHWVRQAPGKGLEWVSAINSG.
..GSTYYADSVK.GRFTISRDNSKNTLSLQMNSLRAEDTAVYYCAK
>IMGT000064|IGHV4-100*01|Macaca mulatta_AG07107|F|V-REGION|790304..790602|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSIS...DDYYWSWIRQPPGKGLEWIGYIYGS.
.GGGTNYNPSLK.NRVTISIDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-115*01|Macaca mulatta_AG07107|F|V-REGION|674569..674870|302 nt|1| | ||100 AA|100+6=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSIS..SSYYYWSWIRQAPGKGLEWIGYIYGG.
.SGSTSYNPSLK.SRVTISRDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-120*01|Macaca mulatta_AG07107|F|V-REGION|643195..643493|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGYSIS...SGYGWSWIRQPPGKGLEWIGYIGGS.
.SGSTNYNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-135*01|Macaca mulatta_AG07107|F|V-REGION|535948..536246|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCTVSGGSIS...GYYYWSWIRQPPGKGLEWIGGIYGN.
.SASTYYNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-138*01|Macaca mulatta_AG07107|F|V-REGION|515409..515704|296 nt|1| | | |98 AA|98+8=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSI....SSNYWSWIRQPPGKGLEWIGRIYGS.
.SGSTSYNPSLT.SRVTISTDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-150*01|Macaca mulatta_AG07107|F|V-REGION|383466..383761|296 nt|1| | | |98 AA|98+8=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSI....SSNYWSWIRQPPGKGLEWIGRIYGS.
.GGSTDYNPSLK.SRVTISTDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-155*01|Macaca mulatta_AG07107|F|V-REGION|306584..306879|296 nt|1| | | |98 AA|98+8=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSI....SGYYWSWIRQPPGKGLEWIGYIGGS.
.SGSTYYNPSLK.SRVTISTDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-159*01|Macaca mulatta_AG07107|F|V-REGION|291932..292226|295 nt|1| | | |98 AA|98+8=106| | |
QLQLQESGP.GLVKPSETLSVTCAVSGGSI....SSSYWSWIRQAPGKGLEWIGYIYGS.
.GSSTNYNPSLK.SRVTLSVDTSKNQLSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-163*01|Macaca mulatta_AG07107|F|V-REGION|266120..266415|296 nt|1| | | |98 AA|98+8=106| | |
QLQLQESGP.GLVKPSETLSLTCAVSGGSI....SSNYWSWIRQPPGKGLEWIGRISGS.
.GGSTDYNPSLK.SRVTISTDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-60*01|Macaca mulatta_AG07107|F|V-REGION|1100956..1101254|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSIS...SSNWWSWIRQPPGKGLEWIGYISGS.
.SGSTYYNPSLK.SRVTISTDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-67*01|Macaca mulatta_AG07107|F|V-REGION|1033288..1033586|299 nt|1| | ||99 AA|99+7=106| | |
QVKLQQWGE.GLVKPSETLSLTCAVYGGSIS...GYYYWSWIRQPPGKGLEWIGYIYGN.
.SASTNYNPSLK.NRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-70*01|Macaca mulatta_AG07107|F|V-REGION|1013645..1013943|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSIS...GGYDWSWIRQPPGKGLEWIGYIYGS.
.SGSTNYNPSLK.NRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-74*01|Macaca mulatta_AG07107|F|V-REGION|980561..980856|296 nt|1| | | |98 AA|98+8=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGASI....SSYWWSWIRQPPGKGLEWIGEINGN.
.SGSTYYNPSLK.SRVTISKDASKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-75*01|Macaca mulatta_AG07107|F|V-REGION|965856..966151|296 nt|1| | | |98 AA|98+8=106| | |
QLQLQESGP.GLVKPSETLSLTCAVSGGSI....SGYYWSWIRQPPGKGLEWIGNIDGN.
.IAGTNYNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-86*01|Macaca mulatta_AG07107|F|V-REGION|875233..875531|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGGSIS...SSNWWSWIRQPPGKGLEWIGRISGS.
.GGSTSDNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-87*01|Macaca mulatta_AG07107|F|V-REGION|860197..860495|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.AVVKPSETLSLTCAVSGGSIS...SSNWWSWIRQSPGKGLEWIGGIYGS.
.GGSTEYNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV4-93*01|Macaca mulatta_AG07107|F|V-REGION|819469..819767|299 nt|1| | ||99 AA|99+7=106| | |
QVQLQESGP.GLVKPSETLSLTCAVSGYSIS...SGYYWGWIRQPPGKGLEYIGYISGS.
.SGSTYYNPSLK.SRVTISKDTSKNQFSLKLSSVTAADTAVYYCAR
>IMGT000064|IGHV5-16*01|Macaca mulatta_AG07107|F|V-REGION|1356265..1356560|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKRPGESLKISCKTSGYSF....TSYWISWVRQMPGKGLEWMGAIDPS.
.DSDTRYNPSFQ.GQVTISADKSISTAYLQWSRLKASDTATYYCAK
>NW_001121239|IGHV5-16*02|Macaca mulatta_17573|F|V-REGION|278688..278983|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKRPGESLKISCKTSGYSF....TSYWISWVRQMPGKGLEWMGAIDPS.
.DSDTRYSPSFQ.GQVTISADKSISTAYLQWSSLKASDTATYYCAK
>IMGT000064|IGHV5-179*01|Macaca mulatta_AG07107|P|V-REGION|102716..103009|294 nt|1| | | |98 AA|98+8=106| | |
EVQLVQPGA.EVKRPGESLRISCKTSGYSF....TSYWISWVRQMPGRELEWMGSIHPG.
.TSDTRYNPSFQ.GQVTISADSSISTVYLQWSSLKASDSAMYYCAR
>NW_001121240|IGHV5-179*02|Macaca mulatta_17573|P|V-REGION|204228..204521|294 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKRPGESLRISCKTSGYSF....TSYWISWVRQMPGRELEWMGSIHPG.
.TSDTRYNPSFQ.GQVTISADSSISTVYLQWSSLKASDSAMYYCAR
>IMGT000064|IGHV5-39*01|Macaca mulatta_AG07107|F|V-REGION|1204988..1205283|296 nt|1| | | |98 AA|98+8=106| | |
EVQLVQSGA.EVKRPGESLRISCKTSGYSF....TSSWISWVRQMPGKGLEWMGSIYPG.
.DSDTRYNPSFQ.GHVTISADKSISTTYLQWSSLKASDTATYYCAK
>IMGT000064|IGHV6-1*01|Macaca mulatta_AG07107|F|V-REGION|1587253..1587557|305 nt|1| | | |101 AA|101+5=106| | |
QVQLQESGP.GLVKPSQTLSLTCAISGDSVS..SNSATWNWIRQSPSRGLEWLGRTYYRS
.KWYNDYAQSVQ.NRISINPDTSKNQFSLQLNSVTPEDMAVYYCAR
>IMGT000064|IGHV7-108*01|Macaca mulatta_AG07107|F|V-REGION|737126..737421|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGA.EVKQPGASVKVSCKASGYTF....TSYGMNWVRQAHGQRLEWMGWINTD.
.TGNPTYAQGFK.ERFTFSMDTSISTAYLQISSLKAEDTAVYYCAR
>IMGT000064|IGHV7-121*01|Macaca mulatta_AG07107|P|V-REGION|639979..640274|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQFGP.EVKQPVASVKVFCEASGYSF....TTYGMNWVRQVPGEGLEWMGWMNTN.
.TGNPTYAQG*T.GRFVFSMDTSVSTVHLQISSLKAEDTAVHYCTR
>IMGT000064|IGHV7-140*01|Macaca mulatta_AG07107|P|V-REGION|501767..502062|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGP.QMKQPGASVKFSCKASGYTF....TSYVTNWVRQAPGQGLE*IGWINTN.
.TGNPMFAQGFT.GWFVFSVDTSVSTAYLQISSLKDEDMAVYCC*R
>IMGT000064|IGHV7-183*01|Macaca mulatta_AG07107|F|V-REGION|80078..80373|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGP.EVKQPGASVKVSCKASGYSS....TTYGMNWVRQAPGQGLEWMGWMNTY.
.TGNPTYAQGFT.ERFVFSMDTSVSTVYLQISSLKAEDTAVYYCAR
>NW_001121240|IGHV7-183*02|Macaca mulatta_17573|ORF|V-REGION|226869..227164|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQSGP.EVKQPGASVKVSCKASGYSF....TTYGMNWVRQAPGQGLEWMGWMNTY.
.TGNPTYAQGFT.ERFVFSMDTSVSTVYLQISSLKGEDTAVYYCAR
>IMGT000064|IGHV7-61*01|Macaca mulatta_AG07107|P|V-REGION|1097724..1098019|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQFGP.EAKQPVASVKVFCKASGYSF....TTYGMNWVRQVPGEGLEWMGWMNTN.
.TGNPIYAQG*T.ERFVFSMDTSVSTVHLQISSLKAEDTAVRYCKR
>IMGT000064|IGHV7-71*01|Macaca mulatta_AG07107|P|V-REGION|1010437..1010732|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQFGP.EVKQPVASVRYFCKASGYGF....TTYGMNWVRQVPGEGLEWMGWMNTN.
.TGNPTYAQG*T.GRFIFSMDTSVSTVHLQLSSLKAEDTAVRYCTR
>IMGT000064|IGHV7-81*01|Macaca mulatta_AG07107|P|V-REGION|919986..920281|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQFGP.EVKQPVASVKVFCKASGYSF....TTYGMNWVRQVPGEGLEWMGWMNIN.
.TGNPMYAQG*T.G*FVFSMDTSVSTAHLQISSLKAEDMAVHYCMR
>IMGT000064|IGHV7-88*01|Macaca mulatta_AG07107|P|V-REGION|856373..856668|296 nt|1| | | |98 AA|98+8=106| | |
QVQLVQFGP.EVKQPVASVKVFCKASGYSF....TTYGMNWVRQVPGEGLEWMGWMNIN.
.TGNPMYAQG*T.GRFVFSMDTSVSTVYLQISSLKAEDTAVHYCMR

</pre>
<hr />





<a href="/genedb/doc">-> IMGT/GENE-DB Documentation</a><br />
<a href="/genedb/">-> IMGT/GENE-DB Query page</a><br />
<a href="/genedb/directlinks">-> IMGT/GENE-DB direct links</a><br />
<a href="/genedb/stats">-> IMGT/GENE-DB statistics</a><br />
<a href="/IMGTgenedbdoc/programversions.html">-> IMGT/GENE-DB program versions</a><br />
<a href="/IMGTgenedbdoc/dataupdates.html">-> IMGT/GENE-DB data updates</a>

<hr />

<p class="menu">
<a href="/">IMGT Home page</a>
| <a href="/IMGTrepertoire/">IMGT Repertoire (IG and TR)</a>
| <a href="/IMGTrepertoireMH/">IMGT Repertoire (MH)</a>
| <a href="/IMGTrepertoireRPI/">IMGT Repertoire (RPI)</a>
| <a href="/IMGTindex/">IMGT Index</a>
| <a href="/IMGTScientificChart/">IMGT Scientific chart</a>
| <a href="/IMGTeducation/">IMGT Education</a>
</p>



<span class="logosFoot">
<a href="https://www.cnrs.fr"><img src="/images/cnrs.png" alt="logo CNRS" style="vertical-align:middle;" /></a>
<a href="https://www.umontpellier.fr/"><img src="/images/logos/universite_mtp.png" alt="Universit&#233; de Montpellier" style="width:60px;vertical-align:middle;" /></a>
<a href="https://ec.europa.eu/info/research-and-innovation_en"><img src="/images/UE.png" alt="logo UE" style="vertical-align:middle;" /></a>
</span>



<p>&#169;  Copyright 1995-2020 IMGT&reg;, the international ImMunoGeneTics information system&reg;
			| <a href="/about/termsofuse.php">Terms of use</a> | <a href="/about/aboutus.php">About us</a> | <a href="mailto:contact-imgt@igh.cnrs.fr">Contact us</a> | <a href="/about/CitingIMGT.php">Citing IMGT</a></p>

<!-- #echo var=$LAST_MODIFIED -->

</body>
</html>

</body>
</html>